process/phases.md

```diff
@@ -45,22 +45,22 @@ Entry requirements:
 During this phase:
 
   1. If the proposal is not already listed, it should be added to the [proposal list](https://github.com/WebAssembly/proposals/blob/main/README.md) at this time.
-  1. A new repository, forking the spec repo, is created by one of the WebAssembly organization administrators, or transfered to the WebAssembly organization by the champion.
+  1. A new repository, forking the spec repo, is created by one of the WebAssembly organization administrators, or transfered to the WebAssembly organization by the champion. See the [how-to](https://github.com/WebAssembly/proposals/blob/main/howto.md) for this step.
   1. The champion will attempt to reach broad consensus in the Community Group.
```

Comment:
Compared to the other points in this section, this one seems vague. Given the language in the point below, is this still necessary? The overview document seems like an artifact or outcome of having consensus.

Reply:
Good question. I believe this is meant to point out that it is an important part of a champion's responsibility to build consensus, not just produce technical artefacts. I think it actually makes sense to call that out somehow. Whether this is the best phrasing I'm not sure.

Reply:
Ah, I see what you mean - yeah I'm not sure if there's a way to convey that well without being too prescriptive of how to build consensus.